assets/js/script.js

```diff
@@ -40,7 +42,36 @@ var input2 = document.getElementById("to");
 // var dropdownTrigger = document.querySelector(".dropdown-trigger");
 
 
+
+
 // --------------- FUNCTIONALITY BELOW ---------------
+//create a script element with a src attribute, add another attribute defer (true)
+
+//  var script = document.createElement('script');
+//          script.src = "https://maps.googleapis.com/maps/api/js?key=" + storedValue + "&libraries=places";
+//         document.body.appendChild(script);
+
+//Create Function that adds the secure API key to the HTML file 
+function apiKeyAdder() {
+            //var apiKeyLink = document.getElementById("api-key");
+          var createdLink = "https://maps.googleapis.com/maps/api/js?key=" + storedValue + "&libraries=places";
+          console.log(createdLink)
+            //apiKeyLink.setAttribute("src", createdLink);
+            //apiKeyLink.setAttribute("defer", true);
+          
+            var s = document.createElement( 'script' );
+            s.setAttribute( 'src', createdLink );
+            s.onload=callback;
+            document.body.appendChild( s );
+
+
+
+
+
+        }
+
+        apiKeyAdder()
```

Comment:
we're required to turn in clean code with proper formatting: please remove the indentation on these two lines. The closing curly bracket should be at the same indentation as where the function begins. This also applies to code that is initiated outside of the function, in this case apiKeyAdder() should not have an indentation